src/harness/reference_models/examples/testutils.py

```diff
@@ -80,3 +84,57 @@ def __call__(self,
         incidence_angles=wf_itm._IncidenceAngles(
             hor_cbsd=bearing_cbsd, ver_cbsd=0, hor_rx=bearing_rx, ver_rx=0),
         internals={})
+
+class FakeInterferenceCalculator(object):
+  """Fake model to calculate the interference for testing.
+
+  It can be used as a fake replacement for computeInterference() module.
+  This fake model internally calls the fake models for Antenna Gain
+  and Propagation Loss calculations.
+
+  interf.computeInterference() = FakeInterferenceCalculator()
+
+  Attributes:
+    ant_gain: Effective antenna gain from CBSD towards the protected entity
+              and vice versa. Default value is configured for interference
+              calculation
+    antenna_height: Antenna height considered for incidence angle calculation
+    reference_bw: Aggregate interference is calculated in reference bandwidth
+  """
+
+  def __init__(self,
```

Comment:
Would have been nice to be able to pass a FakePropagationPredictor to the constructor, that is saved
in self.propag_predictor:
``` 
def __init__(self,
                  ant_gain=10,
                  reference_bw=5.e6,
                  propag_predictor=None):
      self.antenna_height = antenna_height
      self.ant_gain = ant_gain
      self.reference_bw = reference_bw
      if propag_predictor is None:
         propag_predictor = FakePropagationPredictor()
      self.propag_predictor = propag_predictor
```